tools/update-copyright-notices.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eu
+
+YEAR=$(date +%Y)
+echo "Updating global copyright strings..."
+sed -re 's/(Copyright.*20[0-9]{2}-)[0-9]{4}/\1'$YEAR'/g' -i src/res/translation/*.ts src/util.cpp src/aboutdlgbase.ui
```

Comment:
Although a bit picky (and we still have a bunch of years left): but could you use a less specific regex in terms of not specifying the prefix 20 for the year?

Reply:
Done, although I've kept the `2`. I really want to avoid that the script touches an unintended place with its modifications.

Reply:
Understandable 😀.